display_list/dl_builder.cc

```diff
@@ -44,10 +65,15 @@ void* DisplayListBuilder::Push(size_t pod, Args&&... args) {
   size_t size = SkAlignPtr(sizeof(T) + pod);
   FML_CHECK(size < (1 << 24));
```

Comment:
Does this assertion make sense now?

Reply:
I'm not certain the origin of this check.

Reply:
Me either, it might be a carry over from SkLiteDL days?

Reply:
Ooopa. No! I just rememered!

It is necessary because the size field is 24-bits.

Reply:
Well, I'll add it back when I get back lol

Reply:
It's never fired AFAIK, so maybe this should be the impetus that gets me to rewrite builder to not use the size fields any more and produce the offset vector up front...

Reply:
AFAIK this check could only fire If I tried to encode something that required allocating more than 1 << 24 bytes right?